arviz/utils.py

```diff
@@ -664,3 +665,35 @@ def credible_interval_warning(credible_interval, hdi_prob):
 
     hdi_prob = credible_interval
     return hdi_prob
+
+
+def html_templates():
+    html_template = """
+            <div>
+              <div class='xr-header'>
+                <div class="xr-obj-type">arviz.InferenceData</div>
+              </div>
+              {}
+            </div>
+            """
+    element_template = """
+            <ul class="xr-sections">
+            <li class = "xr-section-item">
+                  <input id="xr-dataset-coord_{0}" class="xr-section-summary-in" type="checkbox">
```

Comment:
given that the group name is added to the `id` I am guessing the id is arbitrary and the class is what defines the format used. Is that so? If so, can this be renamed so it becomes more clear what is happening? 

If I am not mistaken, in `{1}` we put the whole html_repr of the xarray dataset corresponding to the group at hand, so having xr, dataset and coord but no az, idata nor group in id section is confusing.

Reply:
Yes, the id is arbitrary. I left it as it is when I first tinkered with xarray's code and forgot to change it. I've changed it.